frontend/main.js

```diff
@@ -9,6 +9,7 @@ import { GIMessage } from '~/shared/domains/chelonia/chelonia.js'
 import { CONTRACT_IS_SYNCING } from '~/shared/domains/chelonia/events.js'
 import './controller/namespace.js'
 import './controller/actions/index.js'
+import './controller/backend.js'
```

Comment:
Kinda shocked this got removed somehow and nobody noticed until now

Reply:
Yea, me too